index.html

```diff
@@ -4098,29 +4099,53 @@ <h5>Existing implementations</h5>
 This provides added flexibility, at the expense of increasing the complexity of code that uses these libraries.
 </p>
 </dd>
+<dt>d3-geographies-api</dt>
+<dd>
+<p><a>supported, with limitations</a>:
+The <var>d3-geo</var> module supports vector data in the form of either GeoJSON or TopoJSON.
+It renders this data to either an SVG or Canvas element.
+It requires the developer to implement support for interaction using other d3 modules, such as <var>d3-zoom</var>
+and <var>d3-transition</var>.
+</p>
+<p>
+By utilising the <var>d3-tile</var> module, tiled raster data can be combined with vector map data.
+</p>
+</dd>
 </dl>
 
 <h5>Supported use cases</h5>
 <ul data-ucr-role="capability-uses"><!-- auto-generated from use case required capabilities --></ul>
 
 <h5>Uses beyond mapping</h5>
 <p>
-Leaflet, OpenLayers, and MapBox allow for the creation of layers displaying non-geographical imagery.
+Leaflet, OpenLayers, and MapBox allow for the creation of layers displaying non-geographical raster tiles.
 </p>
 
 <h5>Related web specifications</h5>
```

Comment:
Since this use case is specifically about creating a map from script, maybe rewrite to focus on that specifically? What do other APIs look like for initializing embedded content? This could include a short summary of basic DOM methods ("For most complex web content, such as audio or video players or iframe documents, initializing the content with native DOM methods means creating the individual elements of the DOM tree and appending them together, or parsing a block of markup text to generate the same tree."), as well as more specialized APIs, like the [`new Image()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image) function as an alternative to creating an `img` element.

Notes on referencing (should have caught this on the previous PR for this section):

- Use the triple bracket shorthand (`[[[HTML]]]`) in the middle of a sentence to get the link to the spec & it's full name inserted.
- Use `<a data-cite="HTML">link text</a>` if you want the sentence text to be something other than the name of the spec.
- Use the double bracket shorthand (`[[HTML]]`) at the end of the paragraph, to get a footnote-like link to the references section.

See [the ReSpec User's Guide for more](https://github.com/w3c/respec/wiki/ReSpec-Editor's-Guide#references)

Reply:
Thanks, that’s helpful - I’ll work on rewriting this and redrafting the other capabilites I have in draft to be more clearly focused on those aspects.

And I’ll review the ReSpec guide and sort out the references :-)